.github/workflows/push-dockerhub-readme.yml

```diff
@@ -0,0 +1,23 @@
+name: push readme to dockerhub
```

Comment:
I think I would better like to have it back in the `build-images.yml`, mostly to be more resource efficient (avoid having to spin a new VM and git clone once more)

Reply:
It was a seperated job
so it always create a new context on another machine

Reply:
Can't we just add the steps to the existing job?

Reply:
(It would be fine to react to the same events we already have)

Reply:
But good point, with your current suggestion, this job might not have to run that often.
Let's keep it that way then.